tripy/tripy/frontend/module/module.py

```diff
@@ -106,7 +106,7 @@ def __setattr__(self, name: str, value: Any) -> None:
 
         if isinstance(value, List) or isinstance(value, Dict):
             container = value if isinstance(value, List) else value.values()
-            if _contains_types(container, [Parameter, Module]) and not _is_homogeneous_container(container):
+            if _contains_types(container, [Parameter, Module]) and not _is_homogeneous_container(container, [Module, Parameter]):
```

Comment:
@akhilg-nv lets add a unit test for this case.

Reply:
done, please check.